src/ReflectionType.php

```diff
@@ -93,7 +94,11 @@ public static function convertToDisplayType(BaseReflectionType $type)
         $displayType = ltrim($displayType, '\\');
 
         if ($type->allowsNull()) {
-            $displayType .= ' or NULL';
+            if (! $type instanceof ReflectionUnionType) {
+                $displayType = '?' . $displayType;
+            } else {
+                $displayType = '|null';
+            }
```

Comment:
@lisachenko this is the BC break for display type, as original `ReflectionParameter::__toString()` on php 8.0 now uses `?` instead of `or NULL` for parameter type information.

Reply:
I guess, it is ok here, we follow logic in the original ReflectionAPI. So, I'm ok to keep this in next major version for 8.x